src/View/mainpage/DVHTab.py

```diff
@@ -133,14 +133,60 @@ def plot_dvh(self):
         return fig
 
     def prompt_calc_dvh(self):
-        choice = QtWidgets.QMessageBox.question(self, "Calculate DVHs?", "Would you like to calculate DVHs? This may"
-                                                                         " take up to several minutes on some systems.",
-                                                QtWidgets.QMessageBox.Yes | QtWidgets.QMessageBox.No)
-
-        if choice == QtWidgets.QMessageBox.Yes:
-            progress_window = CalculateDVHProgressWindow(self, QtCore.Qt.WindowTitleHint | QtCore.Qt.WindowCloseButtonHint)
-            progress_window.signal_dvh_calculated.connect(self.dvh_calculation_finished)
-            progress_window.exec_()
+        """
+            Windows displays buttons in a different order from Linux. A check for
+            platform is performed to ensure consistency of button positioning across
+            platforms.
+        """
+        if platform.system() == "Linux":
+            choice = QtWidgets.QMessageBox.question(self, "Calculate DVHs?",
+                                                    "RTSTRUCT and RTDOSE datasets identified. Would you "
+                                                    "like to calculate DVHs? (This may take up to "
+                                                    "several minutes on some systems.)",
+                                                    QtWidgets.QMessageBox.Yes | QtWidgets.QMessageBox.No)
+
+            if choice == QtWidgets.QMessageBox.Yes:
+                self.signal_advise_calc_dvh.emit(True)
+            else:
+                self.signal_advise_calc_dvh.emit(False)
+        else:
+            stylesheet_path = ""
+
+            # Select appropriate style sheet
+            if platform.system() == 'Darwin':
+                stylesheet_path = "res/stylesheet.qss"
+            else:
+                stylesheet_path = "res/stylesheet-win-linux.qss"
+
+            # Create a message box and add attributes
+            mb = QtWidgets.QMessageBox()
+            mb.setIcon(QtWidgets.QMessageBox.Question)
+            mb.setWindowTitle("Calculate DVHs?")
+            mb.setText("RTSTRUCT and RTDOSE datasets identified. Would you "
+                       "like to calculate DVHs? (This may take up to several "
+                       "minutes on some systems.)")
+            button_no = QtWidgets.QPushButton("No")
+            button_yes = QtWidgets.QPushButton("Yes")
+
+            """ We want the buttons 'No' & 'Yes' to be displayed in that exact order. QMessageBox displays buttons in
+                respect to their assigned roles. (0 first, then 0 and so on) 'AcceptRole' is 0 and 'RejectRole' is 1 
+                thus by counterintuitively assigning 'No' to 'AcceptRole' and 'Yes' to 'RejectRole' the buttons are 
+                positioned as desired.
+            """
+            mb.addButton(button_no, QtWidgets.QMessageBox.AcceptRole)
```

Comment:
That very similar code is being repeated suggests one might want to create a function that gets shared / reused.

Reply:
Yes, you are correct, however, I could not find a simple clean way of creating a single function that wouldn't require passing references from class to class which would get a little messy.
I suspect this is why the code was written the way it originally was.
 
My initial solution was to make the function static from within DVHTab that the other objects can reference, however, QMessageBox requires a reference to an instance of the parenting classes QWidget class. This cannot be accessed if the function were static.
From my understanding, this is where the styling of the buttons and such comes from. 
I could create a new QMessageBox object and assign the stylesheet as I have done for the windows correction, however on Linux versions, the style differentiates from what is currently shown on master for an unknown reason. 
I did not want to change too much of what is currently working. Please if you have any suggestions they are warmly welcome.

Reply:
OK.  You examined the problem and there wasn't a reasonable solution that presented itself.  Fair enough!